opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -1047,6 +1047,15 @@ namespace Opm
             return productivity_index_;
         }
 
+        double productivityIndex(const std::string& well) const {
+            auto well_iter = this->wellMap().find(well);
+            if (well_iter == this->wellMap().end())
+                throw std::logic_error("Tried to get PI for unknown well: " + well);
+
+            return this->productivity_index_[well_iter->second[0]];
```

Comment:
Just for future reference as you develop this feature further: The `productivity_index_` vector has [`pu.num_phases`](https://github.com/OPM/opm-simulators/blob/0056d4e9c124da01123cd9b3905e7c0970a76423/opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp#L173) elements per well.  The one that's supposed to be used in the `WELPI` keyword is the one [pertaining](https://github.com/OPM/opm-simulators/blob/0056d4e9c124da01123cd9b3905e7c0970a76423/opm/simulators/wells/BlackoilWellModel_impl.hpp#L2572-L2601) to the well's preferred phase.

Reply:
Thanks!